doc/files.md

```diff
@@ -56,7 +56,8 @@ Subdirectory       | File(s)               | Description
 `./`               | `fee_estimates.dat`   | Stores statistics used to estimate minimum transaction fees and priorities required for confirmation
 `./`               | `guisettings.ini.bak` | Backup of former [GUI settings](#gui-settings) after `-resetguisettings` option is used
 `./`               | `mempool.dat`         | Dump of the mempool's transactions
-`./`               | `onion_private_key`   | Cached Tor onion service private key for `-listenonion` option
+`./`               | `onion_private_key`   | Obsolete: cached Tor onion service private key for `-listenonion` option. Was used for TORv2 services. Bitcoin Core does not need this file and ignores it starting from version 0.21.
```

Comment:
1550aa8 nits, s/TORv2/Tor v2/ and s/does not need/no longer needs/ (though if you move it to the legacy section, use the "replaced by" format instead)

Reply:
Done - "replaced by ..."